packages/react-cosmos-apollo-proxy/src/fixtureLink.js

```diff
@@ -11,19 +11,47 @@ export function createFixtureLink({ apolloFixture, cache, fixture }) {
           observer.error(failWith);
         }
 
-        observer.next({
-          data:
-            typeof resolveWith === 'function'
-              ? resolveWith({ cache, variables, fixture })
-              : resolveWith.data
-                ? resolveWith.data
-                : resolveWith,
-          errors:
-            (typeof resolveWith !== 'function' && resolveWith.errors) ||
-            undefined
-        });
+        let result;
```

Comment:
Any reason to hoist the `result` variable here?

Reply:
Missed refactor - I'll change it. Along with trying to flatten the function down a bit more.